### DISCUSSION TITLE
vue3 script setup - Table "Body slots"

### DISCUSSION THREAD

**Discussion by tinohager** [quasarframework/quasar#15592] [General - Components / Directives / etc]
If you use the script setup functionality in vue3 and use props there is a conflict would it make sense to change the name of the table props?

```js
<script setup>
const props = defineProps({
  foo: String
})
</script>
```

```html
<template>
  <q-table
    :rows="items"
  >
    <template #body-cell-name="props">
   </template>
</q-table>
```

> WARNING in TestTable.vue:118:32
vue/no-template-shadow: Variable 'props' is already declared in the upper scope.
    116 |       </q-input>
    117 |     </template>
  > 118 |     <template #body-cell-name="props">

- https://vuejs.org/api/sfc-script-setup.html#defineprops-defineemits
- https://quasar.dev/vue-components/table#body-slots

**Comment by metalsadman** [comment]
You can name it whatever you want, scope/myprops etc...
